.eslintrc.js

```diff
@@ -2,10 +2,13 @@ module.exports = {
     extends: [
         "@khanacademy",
     ],
-    plugins: ["import", "jest", "promise", "monorepo"],
+    plugins: ["import", "jest", "promise", "monorepo", "disable"],
     settings: {
-        react: {
-            version: "detect",
+        "eslint-plugin-disable": {
+            paths: {
+                react: ["./*.js", "src/*.js"],
+                "jsx-a11y": ["./*.js", "src/*.js"],
+            },
```

Comment:
What's adding rules for `react` and `jsx-a11y` that we have to disable them here?